src/components/chart/ha-network-graph.ts

```diff
@@ -188,6 +189,25 @@ export class HaNetworkGraph extends SubscribeMixin(LitElement) {
       label: {
         show: showLabels,
         position: "right",
+        formatter: (params: any) => {
+          const node = params.data;
```

Comment:
```suggestion
        formatter: (params: CallbackDataParams) => {
          const node = params.data as Record<string, unknown>;
```
Still not full typing but a little better